support-frontend/app/services/pricing/PriceSummaryService.scala

```diff
@@ -12,22 +12,28 @@ import org.joda.time.Months
 
 import scala.math.BigDecimal.RoundingMode
 
-class PriceSummaryService(promotionService: PromotionService, catalogService: CatalogService)
-    extends TouchpointService {
+class PriceSummaryService(
+    promotionService: PromotionService,
+    defaultPromotionsService: DefaultPromotionsService,
+    catalogService: CatalogService,
+) extends TouchpointService {
   private type GroupedPriceList = Map[(FulfilmentOptions, ProductOptions, BillingPeriod), Map[Currency, PriceSummary]]
 
   def getPrices[T <: Product](
       product: T,
       promoCodes: List[PromoCode],
       readerType: ReaderType = Direct,
   ): ProductPrices = {
-    val promotions = promotionService.findPromotions(promoCodes)
+    val defaultPromos = getDefaultPromoCodes(product)
```

Comment:
```suggestion
    val defaultPromoCodes = getDefaultPromoCodes(product)
```
Just to make explicit the difference between a promotion and a promoCode. This probably applies elsewhere as well